### PR TITLE
Updates to validation errors doc

### PR DESCRIPTION
**Pull request by Meggin**
Note: I'm currently working on intro and validation tables in the amphtml repo as well. This doc needs to go live so that the links in the specs work.

### CODE REVIEW

File: _reference/validation_errors.md [hunk 1]
```diff
@@ -640,10 +638,12 @@ through an online CSS validator, for example,
   </tr>
 </table>
 
-Similar to CSS_SYNTAX error,
-this error occurs when you've CSS syntax errors
-in the specified tag.
-This error tells you the specific rule that's invalid,
+This error refers to the @-rules within CSS,
+for which AMP only allows a handle
```
Comment (Gregable) [inline _reference/validation_errors.md]: did you mean handful? Unsure what 'handle' means in this context.

File: _reference/validation_errors.md [hunk 2]
```diff
@@ -640,10 +638,12 @@ through an online CSS validator, for example,
...
+(see also the [AMP specification](/docs/reference/spec.html)).
+For example, <code>@import</code> isn't allowed.
+The validation error specifically
+tells you the rule that's invlaid,
```
Comment (Gregable) [inline _reference/validation_errors.md]: invalid

**Comment by Gregable** [comment]
Otherwise looks good to me.

File: _reference/validation_errors.md [hunk 3]
```diff
@@ -209,12 +208,11 @@ all attributes with a prefix of `"data-"` are also whitelisted.
...
+CDATA is the content data between a start and end HTML tag
 and is currently evaluated with both whitelists and blacklists.
-Tags with mandatory cdata include:
+Tags with mandatory CDATA include:
 
-* `<style amp-boilerplate>body{...}</style>`
-* `<noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>`
+* `<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>`
```
Comment (cramforce) [inline _reference/validation_errors.md]: nit: Maybe use this form
```markdown
 ``.`html
  <html>
  `.``
```
For better syntax highlighting? (Remote dots; just added for encoding reasons).

**Comment by Meggin** [comment]
PTAL. Implemented review comments and added five new errors.

**Comment by cramforce** [comment]
Lets merge this. Please squash commits.

**Pull request merged**